daemon/k8s_watcher.go

```diff
@@ -695,7 +694,7 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 	go namespaceController.Run(wait.NeverStop)
 	d.k8sAPIGroups.addAPI(k8sAPIGroupNamespaceV1Core)
 
-	endpoint.RunK8sCiliumEndpointSyncGC()
+	k8s.RunK8sCiliumEndpointSyncGC()
```

Comment:
k8s.RunK8s stutters a bit, but not a biggie.

Reply:
Good catch, will fix in next upload.